src/app/app.routes.ts

```diff
@@ -11,7 +11,7 @@ import {routes as networkRoutes} from "./pages/network/network.routes";
 
 export const routes: Routes = [
   {path: '', loadComponent: () => import('./pages/home/home.component').then(c => c.HomeComponent)},
-  {path: 'news', data: {title: $localize`News`}, children: newsRoutes},
+  {path: 'blog', data: {title: $localize`Blog`}, children: blogRoutes},
```

Comment:
Please add a redirect to not break existing links (e.g. on social media)

https://github.com/tlm-solutions/kindergarten/blob/master/src/app/app.routes.ts#L79

Reply:
fixed?

Reply:
use a redirect not clone the content